packages/dart/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@
 * General improvements
 * Updated dependencies
 
+### Refactoring
```

Comment:
Could you move this to the top under a new version 3.1.1?
And rename the title to "Refactor".